.gitignore

```diff
@@ -0,0 +1,14 @@
+.gradle
```

Comment:
quick discussion on the naming of this repo, 
where did `service-authorisation-access-client` come from?

The api this talks to is called `service-auth-provider-api` it is also known as `service-2-service-auth` and `s2s`

I would suggest:
`service-2-service-java-client`
or
`service-auth-provider-java-client`

Thoughts anyone?

Reply:
I personally like the second one more

Reply:
I'm happy with that, @dharmendrak ?